src/locale/zh-cn.js

```diff
@@ -12,16 +12,16 @@ export default moment.defineLocale('zh-cn', {
     weekdaysShort : '周日_周一_周二_周三_周四_周五_周六'.split('_'),
     weekdaysMin : '日_一_二_三_四_五_六'.split('_'),
     longDateFormat : {
-        LT : 'Ah点mm分',
-        LTS : 'Ah点m分s秒',
-        L : 'YYYY-MM-DD',
+        LT : 'HH:mm',
+        LTS : 'HH:mm:ss',
+        L : 'YYYY年MMMD日',
         LL : 'YYYY年MMMD日',
-        LLL : 'YYYY年MMMD日Ah点mm分',
-        LLLL : 'YYYY年MMMD日ddddAh点mm分',
-        l : 'YYYY-MM-DD',
+        LLL : 'YYYY年MMMD日 HH:mm',
```

Comment:
@ibread Sorry, I wasn't too clear about what I had meant. Let me try clear bit.

There are a few tokens that can be used to format a moment based on its locale:
- LT
- LTS
- L
- LL
- LLL
- l
- ll
- lll
- llll

The lowercase version is intended to be the shortened version of its uppercase counterpart.

![screenshot from 2016-11-18 19-04-00](https://cloud.githubusercontent.com/assets/2544489/20429128/b0ee5aa8-adc7-11e6-9f90-30c996d12be7.png)


I am not against using `HH:mm` for `lll` and `llll`, that looks good to me.
But as for `LLL` and `LLLL`, I think may be using `HH时mm分` is better. 

For Example:

```
Date: 2006-01-02 15:04:05
lll: 2006年1月2日 15:04
LLL: 2006年1月2日 15时4分 
```

```
Date: 2006-01-02 16:00:00
lll: 2006年1月2日 16:00
LLL: 2006年1月2日 16时
```

I am referencing `GB/T 15835-2011` because it provides some cases of writing DateTime in Chinese. 
I did not mean we _should_ use format like `2006年1月2日 15时4分`, I mean we _could_ use that format and it is not uncommon.

Like Microsoft Excel, we should provide both options and let user choose what they want.  

![screenshot from 2016-11-18 19-09-38](https://cloud.githubusercontent.com/assets/2544489/20429123/a45946b8-adc7-11e6-8b57-6875950d1787.png)